packages/react-instantsearch/src/__tests__/common-widgets.test.tsx

```diff
@@ -372,6 +373,19 @@ const testSetups: TestSetupsMap<TestSuites> = {
   createMenuSelectWidgetTests() {
     throw new Error('MenuSelect is not supported in React InstantSearch');
   },
+  createDynamicWidgetsWidgetTests({ instantSearchOptions, widgetParams }) {
+    render(
+      <InstantSearch {...instantSearchOptions}>
+        <div className="ais-DynamicWidgets">
+          <DynamicWidgets {...widgetParams}>
+            <RefinementList attribute="brand" />
+            <Menu attribute="category" />
```

Comment:
let's put a hierarchical menu in the static widgets too? that way it tests the "attributes" fetching